app/controllers/post.js

```diff
@@ -1,16 +1,18 @@
 const userModel = require('../models/User')
 const bcrypt = require('bcrypt')
 const jwt = require('jsonwebtoken')
+const PostModel = require('../models/Post')
 
 module.exports = {
-  create: function (req, res, next) {
-    userModel.create({ name: req.body.name, email: req.body.email, password: req.body.password }, function (err, result) {
-      if (err) {
-        next(err)
-      } else {
-        res.json({ status: 'success', message: 'User added successfully!!!', data: null })
-      }
-    })
+  create: async (req, res, next) => {
+    const post = new PostModel(req.body)
+    try {
+      await post.save()
```

Comment:
@rak-shit I think you should also handle the image upload, then only the image URL will be generated.
Please handle image upload.

Reply:
Okay. I will check it. :)